v1/api/swagger/components/schemas/NewNetwork.yaml

```
@@ -23,3 +23,9 @@ properties:
       If not specified, the network uses the default IP pool allocated from the 
       hoster/project IP Pool factory and if specified, a new custom IP Pool will 
       be created and used for IP allocations.
+  VLAN:
+    type: integer
+    description: VLAN ID of the network
```

Is this used for Create only, or for both Create & Get? If it is for Create only, make it more verbose like the following. If it is used for Get as well, then what you have is good enough. Same for VNI below.
```suggestion
    description: optional VLAN ID of the network. If not specified, it is allocated from reserved pool.
```